2015/music-and-measure-theory/german/sentence_translations.json

```diff
@@ -218,7 +218,7 @@
  {
   "input": "This means that if you take a harmonious interval, like a fifth, the ratio of frequencies when played on a piano will not be a nice rational number like you expect, in this case 3 halves, but will instead be some power of the 12th root of 2, in this case 2 to the 7 over 12, which is irrational, but very close to 3 halves.",
   "model": "nmt",
-  "translatedText": "Das heißt, wenn Sie ein harmonisches Intervall wie eine Quinte nehmen, ist das Verhältnis der Frequenzen beim Spielen auf einem Klavier nicht wie erwartet eine schöne rationale Zahl, in diesem Fall drei Hälften, sondern eine Potenz des 12. Grundtons von 2, in diesem Fall 2 zu 7 über 12, was irrational ist, aber sehr nahe an 3 Hälften liegt.",
+  "translatedText": "Das heißt, wenn Sie ein harmonisches Intervall wie eine Quinte nehmen, ist das Verhältnis der Frequenzen beim Spielen auf einem Klavier nicht wie erwartet eine schöne rationale Zahl, in diesem Fall drei Halbe, sondern eine Potenz der 12. Wurzel von 2, in diesem Fall 2 hoch 7 geteilt durch 12, was irrational ist, aber sehr nahe an 3 Halben liegt.",
```

Comment:
in diesem Fall 2 hoch 7 geteilt durch 12, dies ist eine irrationale Zahl, die aber sehr nahe an 3 Halben liegt.